source/hardware/controllers/hercules_djcontrol_inpulse_500.rst

```diff
@@ -0,0 +1,231 @@
+Hercules DJControl Inpulse 500
+==============================
+
+-  `Manufacturer’s product page <https://web.archive.org/web/20241013231548/https://www.hercules.com/en-us/product/djcontrol-inpulse-500/>`__
+-  `Manufacturer’s support and downloads page <https://support.hercules.com/en/product/djcontrolinpulse500-en//>`__
+-  `Forum thread <https://mixxx.discourse.group/t/hercules-djcontrol-inpulse-500/19739/101/>`__
+
+.. versionadded:: 2.6.0
+
+Compatibility
+-------------
+
+This controller is a class compliant USB MIDI and audio device, so it
+can be used without any special drivers on GNU/Linux, Mac OS X, and
+Windows. However, if you wish to use the :ref:`ASIO sound API <preferences-sound-api>`
+under Windows, please install the latest driver package available from
+the `Support
+page <https://support.hercules.com/en/product/djcontrolinpulse500-en//>`__.
+
+Sound card setup
+----------------
+
+This controller has built-in 4 channel output sound card, with MAIN
+output (RCA) and HEADPHONE output (3.5mm jack).
```

Comment:
I think this should be:

with MAIN output (2x RCA or 2x unbalanced mono 6.35mm jack) and HEADPHONE output (stereo 3.5mm jack or stereo 6.35mm jack)